Makefile

```diff
@@ -157,7 +157,7 @@ enter:
 .PHONY: os
 os:
 	@echo -e "\n---> Making Planet/OS (Debian) Docker image...\n"
-	$(MAKE) -e BUILDIMAGE=$(PLANET_OS_IMAGE) DOCKERFILE=os.dockerfile make-docker-image
+	$(MAKE) -e BUILDIMAGE=$(PLANET_OS_IMAGE) DOCKERFILE=os.dockerfile EXTRA_ARGS="--pull" make-docker-image
```

Comment:
Can we make it overridable from command line/environment? I tend to avoid frequent image updates during development as they can take a lot of time.

Reply:
@a-palchikov done!